config/tech-docs.yml

```
@@ -1,16 +1,17 @@
 # See the available options: https://tdt-documentation.london.cloudapps.digital/configure_project/global_configuration/#configure-your-documentation-site
 
 # Host to use for canonical URL generation (without trailing slash)
-host: https://ministryofjustice.github.io/template-documentation-site
+host: https://runbooks.data-catalogue.service.justice.gov.uk
 
 # Header-related options
 show_govuk_logo: true
```

```suggestion
show_govuk_logo: false
```

Not sure we meet the criteria for using govuk branding for this
https://github.com/alphagov/tech-docs-template?tab=readme-ov-file#tech-docs-template

> You’re welcome to use the template even if your service isn’t considered part of GOV.UK, but your site or  service must not:
>
> identify itself as being part of GOV.UK
> use the crown or GOV.UK logotype in the header
> use the GDS Transport typeface
> suggest that it’s an official UK government website if it’s not